pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -104,49 +188,19 @@ func (pl *VolumeZone) Filter(ctx context.Context, _ *framework.CycleState, pod *
 		return nil
 	}
 
+	state, err := getStateData(cs)
+	if err != nil {
+		return framework.AsStatus(err)
```

Comment:
if it's not found, calculate here.

This is necessary because someone might have configured a scheduler with just the Filter plugin. This should continue to work. Similar to this:

https://github.com/kubernetes/kubernetes/blob/70617042976dc168208a41b8a10caa61f9748617/pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go#L154-L159